lib/modules/dosomething/dosomething_campaign/dosomething_campaign.features.field_instance.inc

```diff
@@ -1685,6 +1685,55 @@ function dosomething_campaign_field_default_field_instances() {
     ),
   );
 
+  // Exported field_instance: 'node-campaign-field_on_mobile_app'
+  $field_instances['node-campaign-field_on_mobile_app'] = array(
+    'bundle' => 'campaign',
+    'default_value' => array(
+      0 => array(
+        'value' => 0,
+      ),
+    ),
+    'deleted' => 0,
+    'description' => 'Check this for the campaign to appear on the mobile apps.',
```

Comment:
I thought the feature pulled out those strings for translation down here: https://github.com/DoSomething/dosomething/pull/4360/files#diff-81d7f21ce5de35802663f0371af1d1d1R3082